coarse/models/errors.py

```
@@ -1,13 +1,40 @@
 from enum import Enum
-from typing import List
-from pydantic import BaseModel
+from typing import List, Union
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)